template-manager-core/src/main/java/com/sequenceiq/cloudbreak/template/filesystem/BaseFileSystemConfigurationsView.java

```diff
@@ -54,5 +58,13 @@ public String getType() {
     public String getProtocol() {
         return "";
     }
+
+    public String getIdBrokerIdentityId() {
+        return idBrokerIdentityId;
+    }
+
+    public void setIdBrokerIdentityId(String idBrokerIdentityId) {
```

Comment:
Minor: Are setters really needed in this class? Is not it supposed to be immutable?